soccer_model_msgs/msg/Obstacle.msg

```diff
@@ -2,4 +2,5 @@
 # An Obstacle is defined as anything on the field that could not be classified as
 # one of the known types (such as Robot or Goalpost).
 
-soccer_geometry_msgs/PointWithCovariance point
```

Comment:
Are the soccer_geometry_msgs still needed after this change?

Reply:
It's still being used in Ball.msg

Reply:
Ah right